src/css/site.styl

```diff
@@ -84,9 +84,11 @@ input, select
   width: 100vw
   top: 0
   left: 0
-  z-index: 500
+  z-index: 400
   transition: color 300ms
-
+  nav
+    position: relative
+    z-index: 400
```

Comment:
weird that this is set twice to the same value. I see what is does with the line but think the stack should be more explicit. the header being 100 the block shim being 200 and the navs 300